recycle.xml

```diff
@@ -0,0 +1,210 @@
+<!-- 1 Enrichment 2 Reactors -->
```

Comment:
...and deployinst.

On Wed, Apr 8, 2015 at 3:55 PM, Robert Carlsen <rwcarlsen@gmail.com> wrote:

> I think I'll pull this file into its own PR and we can turn it into the
> canonical cyclus example input file - exampling one of each of the
> archetypes: source, sink, enrichment, fuelfab, separations, reactor,
> storage, and (maybe) conversion.
>
> On Wed, Apr 8, 2015 at 3:17 PM, Matthew Gidden <notifications@github.com>
> wrote:
>
>> In recycle.xml
>> <https://github.com/cyclus/cycamore/pull/309#discussion_r28014362>:
>>
>> > @@ -0,0 +1,210 @@
>> > +<!-- 1 Enrichment 2 Reactors -->
>>
>> this file needs an updated comment block and should be moved either to
>> cycamore/input or more preferably cycamore/tests/input with an
>> associated regression test
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cyclus/cycamore/pull/309/files#r28014362>.
>>
>
>